ui/app/templates/components/masked-input.hbs

```diff
@@ -15,6 +15,15 @@
     data-test-textarea
     />
 	{{/if}}
+  {{#if allowCopy}}
+    <CopyButton
+      @clipboardText={{value}}
+      @class="copy-button button"
+      data-test-copy-button
+    >
+      <ICon @glyph="copy" aria-hidden="true" @size=16 />
+    </CopyButton>
+  {{/if}}
```

Comment:
Do you remember why we put these after the input? I was looking at that the other day and couldn't remember haha.

Reply:
was just talking to Joshua about this. our guess is accessibility